packages/suite-desktop-core/e2e/support/testExtends/suiteBaseFixture.ts

```
@@ -0,0 +1,166 @@
+};
+
+// This is the base Suite text fixture containing all the necessary setup and core page object
+// Depending on the project type (desktop or web) it will launch the appropriate environment
+// and provide the necessary page object which is either electron window or web page
+const suiteBaseTest = base.extend<suiteBaseFixture>({
+    startEmulator: true,
+    setupEmulator: true,
+    emulatorStartConf: { model: 'T3T1', wipe: true },
+    emulatorSetupConf: {},
+    /* eslint-disable-next-line no-empty-pattern */
+    url: async ({}, use, testInfo) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)